macros.in

```diff
@@ -1006,6 +1006,34 @@ package or when debugging this package.\
 %_target_vendor		%{_host_vendor}
 %_target_os		%{_host_os}
 
+#==============================================================================
+# ---- compiler flags.
+
+# C compiler flags.  This is traditionally called CFLAGS in makefiles.
+# Historically also available as %%{optflags}, and %%build sets the
+# environment variable RPM_OPT_FLAGS to this value.
+%build_cflags %{optflags}
+
+# C++ compiler flags.  This is traditionally called CXXFLAGS in makefiles.
+%build_cxxflags %{optflags}
+
+# Fortran compiler flags.  Makefiles use both FFLAGS and FCFLAGS as
+# the corresponding variable names.
+%build_fflags %{optflags} %{?_fmoddir:-I%{_fmoddir}}
+
+# Link editor flags.  This is usually called LDFLAGS in makefiles.
+#%build_ldflags -Wl,-z,relro %{?_lto_cflags}
```

Comment:
Why are we commenting this out? The LTO settings won't take effect as is...

Reply:
Because I don't want to set defaults. That's up to distros to do, it's just an example of what might be there.

Reply:
OK